units/provisioning_setup_host.go

```diff
@@ -805,33 +799,25 @@ func mountLinuxVolume(ctx context.Context, env evergreen.Environment, h *host.Ho
 		}))
 	}()
 
-	device, err := getMostRecentlyAddedDevice(ctx, env, h)
+	// wait for the volume to come up on the instance
+	device, err := waitForDevice(ctx, env, h)
 	if err != nil {
-		return errors.Wrap(err, "can't get device info")
+		return errors.Wrap(err, "can't get device")
 	}
 
-	cmd := client.CreateCommand(ctx).Sudo(true)
-	// umount if it's already mounted
+	// we've done this already
```

Comment:
Just for clarity, can we reword "this" to specify that the device is already mounted?

Reply:
done